src/Event/AfterSendEvent.php

```diff
@@ -22,13 +23,20 @@ class AfterSendEvent extends Event
     private $response;
 
     /**
+     * @var RequestInterface
+     */
+    private $request;
+
+    /**
      * Constructor
      *
      * @param ResponseInterface $response
+     * @param RequestInterface  $request
      */
-    public function __construct(ResponseInterface $response = null)
+    public function __construct(ResponseInterface $response = null, RequestInterface $request = null)
```

Comment:
Switching the order makes sense to me. Not sure about them being null. @natebrunette?